README.md

```diff
@@ -8,17 +8,15 @@ Examples can be found in `HL7::Message`.
 The gem's version id can be found in the `HL7::VERSION` constant.
 
 ###  Links
-* Git:          http://github.com/ruby-hl7/ruby-hl7.git
-* Docs:         http://rubydoc.info/gems/ruby-hl7/1.0.3/frames
+* Git:          https://github.com/ruby-hl7/ruby-hl7.git
+* Docs:         https://rubydoc.info/gems/ruby-hl7/
 * Rubygems:     https://rubygems.org/gems/ruby-hl7
-* Google group: http://groups.google.com/group/ruby-hl7
+* Google group: https://groups.google.com/group/ruby-hl7
 
 ### Maintained by:
 
-* http://csinitiative.com
-* http://trisano.org
-* http://github.com/csinitiative/trisano
-* http://onemedical.com
+* https://github.com/csinitiative/trisano
+* https://onemedical.com
```

Comment:
Might also be good to put your GitHub profile link on here. I'm not aware of these orgs ever contributing to this project while I've been involved.

Reply:
Thanks for the suggestion 🙏

Reply:
Done.